Newsletters/Yearn Finance Newsletter #48/Yearn Finance Newsletter #48.md

```diff
@@ -1,135 +1,133 @@
   
-And it doesn't hurt that Andre Cronje is a big fan.
+Desde luego no hace daño tampoco que Andre Cronje sea un gran fan.
 
-The first vaults on Fantom are: yvWFTM, yvUSDC, yvDAI, and yvMIM. We’ve launched support for a native Fantom Iron Bank as well. Head to [beta.yearn.finance](https://beta.yearn.finance) to check them out for yourself.
+Las primeras vaults en Fantom son: yvWFTM, yvDAI y yvMIM. Lanzamos también soporte nativo para el Iron Bank de Fantom. Puedes verlo por ti mismo en [beta.yearn.finance](https://beta.yearn.finance).
 
-So to answer your questions of when Arbitrum, Optimism, Polygon, Avalanche, and others are arriving, there will definitely be more multichain fun coming soon. Ultimately, we’d love to bring Yearn to any chain where we can safely operate and simplify the process of earning yield for users and partners.
+Para responder a tu pregunta, Arbitrum, Optimism, Polygon, Avalanche y otras están disponibles pronto. Finalmente nos encantaría llevar a Yearn a cualquier otra blockchain donde pueda operar con seguridad y así simplificar los procesos de obtención de rendimiento para sus los usuarios y colaboradores.
 
-The Realm expands and we go with it. We want to meet people where they are, including new users and users with smaller deposits. Multichain expansion is a natural way to do this.
+El reino se expande y nosotros con él. Queremos conocer personas donde estén, incluyendo nuevos usuarios con depósitos más pequeños. La expansión multi blockchain es el camino natural para esto.
 
-Got questions or want to learn more? Join us on Telegram or [Discord](https://discord.com/invite/SNPvCpjRET).
+¿Tienes preguntas o quieres aprender más? Únete a nuestro Telegram o [Discord](https://discord.com/invite/SNPvCpjRET).
 
-## Yearn Reaches $5B TVL (Again)
+## Yearn alcanza 5 Billones de dólares en TVL (De nuevo)
 
  ![](image3.png)
 
-Total value locked across all Yearn products has surpassed $5 billion once again, and $1 billion has been added in the last month alone. Yearn is continuing to see parabolic growth among its products with no indication of slowing down. The Iron Bank and v2 vaults are particularly notable components of our exponential growth. You can view and track TVL metrics among our products using [yearn.science](https://yearn.science/). 
+El valor total bloqueado (TVL) a través de todo Yearn ha superado los 5 billones de dólares una vez más y 1 ilion ha sido obtenido en el mes pasado. Yearn continúa viendo crecimiento parabólico en sus productos sin mostrar signos de disminuir el paso. El Iron Bank y la versión V2 son componentes fundamentales de nuestro crecimiento exponencial. Puedes ver y monitorizar las métricas del TVL en todos nuestros productos utilizando [yearn.science](https://yearn.science/). 
```

Comment:
ÁS-IS: El valor total bloqueado (TVL) a través de todo Yearn ha superado los 5 billones de dólares una vez más y 1 ilion ha sido obtenido en el mes pasado. Yearn continúa viendo crecimiento parabólico en sus productos sin mostrar signos de disminuir el paso. El Iron Bank y la versión V2 son componentes fundamentales de nuestro crecimiento exponencial. Puedes ver y monitorizar las métricas del TVL en todos nuestros productos utilizando [yearn.science](https://yearn.science/).


TO-BE: El valor total bloqueado (TVL) a través de todo Yearn ha superado los 5 billones de dólares una vez más y 1 billón ha sido obtenido en el mes pasado. Yearn continúa viendo crecimiento parabólico en sus productos sin mostrar signos de disminuir el paso. El Iron Bank y la versión V2 son componentes fundamentales de nuestro crecimiento exponencial. Puedes ver y monitorizar las métricas del TVL en todos nuestros productos utilizando [yearn.science](https://yearn.science/).